coursier.bzl

```diff
@@ -741,6 +746,22 @@ def make_coursier_dep_tree(
         # https://github.com/coursier/coursier/blob/1cbbf39b88ee88944a8d892789680cdb15be4714/modules/paths/src/main/java/coursier/paths/CoursierPaths.java#L29-L56
         environment = {"COURSIER_CACHE": str(repository_ctx.path(coursier_cache_location))}
 
+    # If we are using Java 9 or higher we can use an argsfile to avoid command line length limits
+    java_path = _java_path(repository_ctx)
+    if java_path:
+        exec_result = _execute(repository_ctx, [java_path, "-version"])
+        if (exec_result.return_code != 0):
+            fail("Error while running java -version: " + exec_result.stderr)
+        if parse_java_version(exec_result.stdout + exec_result.stderr) > 8:
```

Comment:
Is the output of java -version on both stdout and stderr?

Reply:
It looks like it goes to stderr in the two cases I tried but I wasn't sure if that would be the case for all jvm versions.